### ISSUE TITLE
Change default behaviour for conflicts

### ISSUE THREAD

**Issue by pazembrz** [inspirehep/inspire-json-merger#83]
All conflicts should be displayed as changes form head. So if there is conflict on update, then HEAD value should be in `merged` and update should be put as conflict. So when curator reject all conflicts only HEAD values should be left.

**Comment by michamos** [comment]
I don't think you need to set the default to `KEEP_HEAD`. It can stay as it is, but non-ignored conflicts (those not in `conflict_filters`) should have `KEEP_HEAD` always.
